Tests have been ... driven! These fail on `main`, as they should.

```
$ git checkout -b one main
Switched to a new branch 'one'
$ git cherry-pick bbasata/set
[one d135622] hclwrite: unit testing for Body.SetAttributeRaw
 Date: Wed Jul 29 13:11:25 2026 -0400
 1 file changed, 122 insertions(+)
$ go test ./hclwrite
--- FAIL: TestBodySetAttributeTraversal_ReturnsTheAttribute (0.00s)
    --- FAIL: TestBodySetAttributeTraversal_ReturnsTheAttribute/attribute_`one`_is_not_set_to_a_value (0.00s)
        ast_body_test.go:800: got: nil
            want: one=the.loneliest.number
--- FAIL: TestBodySetAttributeRaw_ReturnsTheAttribute (0.00s)
    --- FAIL: TestBodySetAttributeRaw_ReturnsTheAttribute/attribute_`one`_is_not_set_to_a_value (0.00s)
        ast_body_test.go:996: got: nil
            want: one="the loneliest number"
--- FAIL: TestBodySetAttributeValue_ReturnsTheAttribute (0.00s)
    --- FAIL: TestBodySetAttributeValue_ReturnsTheAttribute/attribute_`one`_is_not_set_to_a_value (0.00s)
        ast_body_test.go:1027: got: nil
            want: one="the loneliest number"
FAIL
FAIL    github.com/hashicorp/hcl/v2/hclwrite    0.570s
FAIL
```

--- a/hclwrite/ast_body_test.go
+++ b/hclwrite/ast_body_test.go
@@ -787,7 +787,7 @@ func TestBodySetAttributeTraversal_ReturnsTheAttribute(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			f := must[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
+			f := testFn[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
 
 			traversal := hcl.Traversal{
 				hcl.TraverseRoot{Name: "the"},
@@ -989,7 +989,7 @@ func TestBodySetAttributeRaw_ReturnsTheAttribute(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			f := must[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
+			f := testFn[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
 
 			attr := f.Body().SetAttributeRaw(`one`, TokensForValue(cty.StringVal("the loneliest number")))
 			if attr == nil {
@@ -1020,7 +1020,7 @@ func TestBodySetAttributeValue_ReturnsTheAttribute(t *testing.T) {
 
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			f := must[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
+			f := testFn[*File](t)(ParseConfig([]byte(test.config), "", hcl.Pos{Line: 1, Column: 1}))
 
 			attr := f.Body().SetAttributeValue(`one`, cty.StringVal("the loneliest number"))
 			if attr == nil {
@@ -1857,12 +1857,12 @@ bar {}
 
 }
 
-// must is a convenience function that wraps a function call that returns a
+// testFn is a convenience function that wraps a function call that returns a
 // value and diagnostics.
 //
 // If there are no diagnostics, returns the value.
 // If there are diagnostics, fails the test.
-func must[E any](t *testing.T) func(value E, diags hcl.Diagnostics) E {
+func testFn[E any](t *testing.T) func(value E, diags hcl.Diagnostics) E {
 	t.Helper()
 
 	return func(value E, diags hcl.Diagnostics) E {

